rules/rbe_repo/build_gen.bzl

```diff
@@ -132,24 +139,38 @@ def create_alias_platform(ctx, toolchain_config_spec_name, image_name, name, use
         target = _CC_TOOLCHAIN,
         toolchain_config_repo = ctx.attr.toolchain_config_suite_spec["repo_name"],
     ))
-    _create_platform(ctx, image_name, name, cc_toolchain_target, use_legacy_platform_definition)
+    _create_platform(ctx, image_name, name, cc_toolchain_target, use_legacy_platform_definition, exec_properties)
 
 # Creates a BUILD file with the cc_toolchain and platform targets
-def _create_platform(ctx, image_name, name, cc_toolchain_target, use_legacy_platform_definition):
+def _create_platform(ctx, image_name, name, cc_toolchain_target, use_legacy_platform_definition, exec_properties):
     template = ctx.path(Label("@bazel_toolchains//rules/rbe_repo:BUILD.platform_legacy.tpl")) if use_legacy_platform_definition else ctx.path(Label("@bazel_toolchains//rules/rbe_repo:BUILD.platform.tpl"))
     exec_compatible_with = ("\"" +
                             ("\",\n        \"").join(ctx.attr.exec_compatible_with) +
                             "\",")
     target_compatible_with = ("\"" +
                               ("\",\n        \"").join(ctx.attr.target_compatible_with) +
                               "\",")
+
+    if use_legacy_platform_definition == True and exec_properties:
```

Comment:
you should fail much earlier than here (around https://github.com/bazelbuild/bazel-toolchains/blob/master/rules/rbe_repo.bzl#L956)

Reply:
done